tools/src/test/resources/com/nedap/archie/serializer/adl/openEHR-EHR-CLUSTER.comments.v1.adls

```diff
@@ -0,0 +1,55 @@
+archetype (adl_version=2.0.5; rm_release=1.0.2)
+	openEHR-EHR-CLUSTER.comments.v1.0.0
+
+language
+	original_language = <[ISO_639-1::en]>
+
+description
+	original_author = <
+		["name"] = <"Jelte Zeilstra">
+		["organisation"] = <"Nedap <https://www.nedap.com>">
+	>
+	details = <
+		["en"] = <
+			language = <[ISO_639-1::en]>
+			purpose = <"A very simple cluster archetype for testing comments">
+		>
+	>
+
+definition
+	CLUSTER[id1] matches {	-- Prescription
+		items matches {
+		    ELEMENT[id2] matches {
+		        value matches {
+		            DV_TEXT[id21]
+                }
+		    }
+		    ELEMENT[id3] matches {
+		        value matches {
+		            DV_CODED_TEXT[id31]
+                }
+		    }
+		}
```

Comment:
Although Github does show archetypes weird, in IntelliJ I can see that the indentation of the elements is inconsistent with the rest of the archetype